build.gradle

```diff
@@ -23,19 +23,15 @@ buildscript {
   }
 
   dependencies {
-    // Newer versions of the nebula plugins require gradle-info-plugin 4.0.2 or higher, which in
-    // depends on JGit 5.0.  Conversely, gradle-git relies of JGit 4.0.
-    // Be mindful of potential classpath issues when updating any of these three dependencies.
```

Comment:
We don't have any issues with JGit on running the tasks that Nebula brings in?

Reply:
No. I updated the git plugin to the non-deprecated plugin and all seems fine.